apache/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM ubuntu:18.04
 LABEL maintainer="anil.natha@jpl.nasa.gov"
```

Comment:
I think we should change this so that a support email is used instead of my personal email. Although I don't know which account we should list as the "maintainer".  e.g. support@maap-project.org maybe?